src/glue/jobs/compare_parquet_datasets.py

```diff
@@ -514,6 +820,9 @@ def main():
     if data_types_to_compare:
```

Comment:
I believe in the original design, we included this because we wanted to add a line to the current comparison report if `main` or `staging` doesn't have a particular data type parquet folder and the other does (as an indicator or whether they have a data type to compare against). But yes, we probably don't need to include that anymore/it seems like more overkill.

I don't think we would be parsing the file names from the exports. We could use a function that's like `has_files_in_s3` that just parses and checks that there are parquet files in the data type folder for that namespace. And then remove adding that line to the compare parquet report.

Then yes we can remove `get_data_types_to_compare`, `compare_dataset_data_types`, and get_folders_in_s3_bucket`

Reply:
I opted to just remove the functionality related to getting available data types. It helps clear up a lot of the complexity in `main`